lib/ra10ke/dependencies.rb

```diff
@@ -138,6 +138,43 @@ def print_table(mods)
     end
   end
 
+  def define_task_convert_to_git(*_args)
+    desc "Print outdated forge modules"
+    task :convert_to_git do
+      require 'ra10ke/git_repo'
+      require 'r10k/puppetfile'
+      require 'puppet_forge'
+
+      PuppetForge.user_agent = "ra10ke/#{Ra10ke::VERSION}"
+      puppetfile = get_puppetfile
+      puppetfile.load!
+      PuppetForge.host = puppetfile.forge if puppetfile.forge =~ /^http/
+
+      # ignore file allows for "don't tell me about this"
+      ignore_modules = []
+      if File.exist?('.r10kignore')
+        ignore_modules = File.readlines('.r10kignore').each {|l| l.chomp!}
+      end
+      forge_mods = puppetfile.modules.find_all { |mod| mod.instance_of?(R10K::Module::Forge) && mod.v3_module.homepage_url? }
```

Comment:
What is `homepage_url`? the `project_page` attribute from metadata.json or `source`?

Reply:
So I think this is were some contention might be because homepage_url is a forge api thing and source is a metadata.json thing.  Both can be right and wrong and might even conflict.  Not sure which one to use so I just picked the one from the API.  Even so, it is still confusing and probably not the best.

I should get the metadata source url and compare.